### DISCUSSION TITLE
Replace YAML with CUE

### DISCUSSION THREAD

**Discussion by sim-san** [home-assistant/architecture#756] [General]
I found this new language CUE from Google which could be a replacement for YAML in some future versions. 

> CUE has all the features you wish YAML had: string interpolation, templating, static type checking, a complete package system, and more. And best of all, it can natively import and export YAML and JSON, for maximum compatibility with your existing tooling.

[https://cuelang.org/](https://cuelang.org/)

**Comment by frenck** [comment]
Thanks for the architectural proposal, however, we don't want to switch to CUE.

Thanks! 👍 

../Frenck
